.github/workflows/ci.yml

```diff
@@ -54,12 +54,11 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-go@v5
         with:
-          go-version: 1.22.5
-          cache: false
+          go-version: 1.23.2
```

Comment:
What's the reason for dropping `cache: false`?

Reply:
I was looking at the latest version of packages and this isn't the recommendation. I remember writing this code but I'm not sure why. It seems like it would be better to cache go version when available.

I'm happy to leave this code in though.